src/constants/index.js

```diff
@@ -27,8 +27,14 @@ const HTTP_CODE = {
   INTERNAL_SERVER_ERROR: 500,
 };
 
+const ENV_LIST = {
```

Comment:
```suggestion
const Env = {
```
Енам просто называется с большой буквы, в кебаб кейсе